x/evm/keeper/grpc_query.go

```diff
@@ -508,12 +508,12 @@ func (k Keeper) TraceTx(c context.Context, req *types.QueryTraceTxRequest) (*typ
 		k,
 		baseFee,
 		func(ctx sdk.Context, cfg *EVMConfig, traceConfig *types.TraceConfig) (*core.Message, error) {
-			signer := ethtypes.MakeSigner(cfg.ChainConfig, big.NewInt(ctx.BlockHeight()))
-			tracer, err := newTacer(&logger.Config{}, cfg.TxConfig, traceConfig)
+			signer := ethtypes.MakeSigner(cfg.ChainConfig, big.NewInt(ctx.BlockHeight()), uint64(ctx.BlockTime().Unix())) //#nosec G115
+			tracer, err := newTacer(&logger.Config{}, cfg.ChainConfig, cfg.TxConfig, traceConfig)
 			if err != nil {
 				return nil, status.Error(codes.Internal, err.Error())
 			}
-			cfg.Tracer = tracer
+			cfg.Tracer = tracer.Hooks
```

Comment:
looking at the line below

```
msg, err := core.TransactionToMessage(ethTx, signer, cfg.BaseFee)
```

It seems that 	`SkipNonceChecks` and  `SkipFromEOACheck` are set to `false` but in geth, those are usually set `true`. Not sure if there is any side effect